fabfile/alba.py

```diff
@@ -136,30 +254,43 @@ def create_loop_devices():
 
 local_nodeid_prefix = str(uuid4())
 
-def _asd_inner(port, path, node_id, slow, multicast):
+def _asd_inner(port, path, node_id, slow, multicast,
+               env, restart):
     global local_nodeid_prefix
     cfg_path = path + "/cfg.json"
-    limit = 90 if env['osds_on_separate_fs'] else 99
-    cfg = { 'port' : port,
-          'node_id' : "%s_%i" % (local_nodeid_prefix, node_id),
-          'home' : path,
-          'log_level' : 'debug',
-          'asd_id' : "%i_%i_%s" % (port,node_id, local_nodeid_prefix),
-          'limit' : limit,
-          '__sync_dont_use' : False
-    }
-    if multicast:
-        cfg ['multicast'] = 10.0
-    else:
-        cfg ['multicast'] = None
+    if not restart:
+        limit = 90 if env['osds_on_separate_fs'] else 99
+        asd_id = "%i_%i_%s" % (port,node_id, local_nodeid_prefix)
+        cfg = { 'port' : port,
```

Comment:
We shouldn't set this property when testing tls, otherwise we're not sure it will really be using tls